src/main/java/com/linkedin/kmf/services/JettyService.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2020 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.linkedin.kmf.services;
+
+import com.linkedin.kmf.services.configs.JettyServiceConfig;
+import java.util.Map;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.ResourceHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Jetty server which serves HTML files. (index.html)
+ */
+public class JettyService implements Service {
```

Comment:
This is not the correct way to do this.  The correct way to do this is to create a war file and then deploy the war file into jetty.  No Java code is required, maybe some gradle code.

Reply:
I read your comment below -- as long as testing is in place, I think a web application + UI that displays real-time monitoring metrics is useful for open source users and more or less expected of Xinfra Monitor as shown in some of the series/walkthroughs such as the above. 

I will create an ask to the original requester to add `testing`.

A fourth option would be creating a separate branch that contains Jetty support for HTML and have the user commit to and maintain it.